transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -73,11 +73,14 @@ public int get() throws Exception {
         }
     };
 
+    private static final long AWAKE = -1L;
+    private static final long NONE = Long.MAX_VALUE;
+
     // nextWakeupNanos is:
-    //    -1               when EL is awake
-    //    Long.MAX_VALUE   when EL is waiting with no wakeup scheduled
+    //    AWAKE            when EL is awake
+    //    NONE             when EL is waiting with no wakeup scheduled
     //    other value T    when EL is waiting with wakeup scheduled at time T
-    private final AtomicLong nextWakeupNanos = new AtomicLong(-1L);
+    private final AtomicLong nextWakeupNanos = new AtomicLong(AWAKE);
```

Comment:
is outside the scope of this PR, but why not an `AtomicLongFieldUpdater`?
It would allow to control its layout if profiling shows that's highly contended (and it seems the case 2 me)

Reply:
Sounds like a good followup PR (which is also true for `EpollEventLoop`.

Reply:
Sure I can do it in a follow-on. I used `AtomicLong` here to be consistent with `EpollEventLoop`. That was originally an atomic updater but @normanmaurer suggested changing it to `AtomicLong` in a prior PR here: https://github.com/netty/netty/pull/9605#discussion_r328955800... I am fine with either though!